### PR TITLE
🎉 New Source: Senseforce [low-code CDK]

### PR DESCRIPTION
**Pull request by andnig**
## What
This PR adds the new low-code CDK based Senseforce connector. [Senseforce](https://senseforce.io/?lang=en) is a european IoT company.

## How
This PR uses the lowcode CDK to create a Omnisend connector with the following streams:
- dataset (full-refresh/incremental): Datasets are the basic data models of any Senseforce installation. This connector now allows to export an arbitrary portion of the Senseforce data models.

## Recommended reading order
1. `spec.yaml`
2. `omnisend.yaml`

## 🚨 User Impact 🚨
No changes to existing code, except docs/integrations/Readme.md

## Pre-merge Checklist
Expand the relevant checklist and delete the others.

<details><summary><strong>New Connector</strong></summary>

### Community member or Airbyter

- [X] **Community member?** Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [X] Secrets in the connector's spec are annotated with `airbyte_secret`
- [X] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [ ] Documentation updated
    - [X] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [X] `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
    - [X] `docs/integrations/README.md`
    - [ ] `airbyte-integrations/builds.md`
- [X] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)

</details>

## Tests

<details><summary><strong>Acceptance</strong></summary>

image

</details>

### CODE REVIEW

**Comment by andnig** [comment]
One comment about CLA signing: Unfortunately I used the wrong E-Mail-Adress for the very first commit (I used andreas.nigg@devopsandmore.come instead of .com....). 
Is this still ok or do I need to rebase the whole thing and change the E-Mail?

**Comment by marcosmarxm** [comment]
There are some tutorials here: https://docs.airbyte.com/connector-development/config-based/tutorial/getting-started besides that please point what is your question to help your, or enter our Slack and ask for help in `#hacktoberfest-2022` channel.

**Review by marcosmarxm — CHANGES_REQUESTED** [review]
Hello @andnig, Marcos from Airbyte here 👋 . We received more than 25 new contributions along the weekend. One is yours 🎉  thank so much for! Our team is limited and maybe the review process can take longer than expected. As described in the Airbyte's Hacktoberfest your contribution was submitted before November 2nd and it is eligible to win the prize. The review process will validate other requirements. I ask to you patience until someone from the team review it. 

Because I reviewed some contributions for Hacktoberfest so far I saw some common patterns you can check in advance:
* [ ] Make sure you have added connector documentation to `/docs/integrations/`
* [ ] Remove the file `catalog` from `/integration_tests`
* [ ] Edit the `sample_config.json` inside `/integration_tests`
* [ ] For the `configured_catalog` you can use only `json_schema: {}`
* [ ] Add title to all properties in the `spec.yaml`
* [ ] Make sure the `documentationUrl` in the `spec.yaml` redirect to Airbyte's future connector page, eg: connector Airtable the `documentationUrl: https://docs.airbyte.com/integrations/sources/airtable`
* [ ] Review now new line at EOF (end-of-file) for all files.

If possible send to me a DM in Slack with the tests credentials, this process will make easier to us run integration tests and publish your connector. If you only has production keys, make sure to create a bootstrap.md explaining how to get the keys.

**Comment by andnig** [comment]
> @andnig please request my review again when all points in my previous comment are solved.

@marcosmarxm I think all points are resolved, are they? Sorry if not - would you mind pointing me to the issue?

**Comment by marcosmarxm** [comment]
Hello! I'm going to be out of the office this Friday and won't be able to review your contribution again today, I return next Monday. So far, most contributions look solid and are almost done to be approved. As said in [Chris' comment](https://airbytehq.slack.com/archives/C0450PCSDB2/p1667430678866629) all contributions made before 2-November are eligible to receive the prize and have 2 weeks to merge the contributions. But I ensure next week we're going to have your contribution merged. If you have questions about the implementation you can send them in `#hacktoberfest-2022` in Airbyte's Slack.

Sorry the inconvenience and see you again next week, thank you so much for your contribution!

**Review by marcosmarxm — CHANGES_REQUESTED** [review]
Some comments, do you have test account to share to run tests?

File: airbyte-integrations/connectors/source-senseforce/source_senseforce/senseforce.yaml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+  requester:
+    # url_base: "http://localhost:8080"
+    url_base: "{{ config['backend_url'] }}"
+    http_method: "POST"
+    request_options_provider:
+      request_body_data: |
+          [{"clause": {"type": "timestamp", "operator": 10, "parameters": 
+              [{"value": {{ stream_slice['start_time'] | int * 1000 }} },
+               {"value": {{ stream_slice['end_time'] | int * 1000 + (86400000 - 1) }} }
```
[the remark this answers is not in the document]
Comment (andnig) [reply airbyte-integrations/connectors/source-senseforce/source_senseforce/senseforce.yaml:16]: It fetches all the data from start_time

File: airbyte-integrations/connectors/source-senseforce/source_senseforce/schemas/TODO.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# TODO: Define your stream schemas
+Your connector must describe the schema of each stream it can output using [JSONSchema](https://json-schema.org). 
+
+You can describe the schema of your streams using one `.json` file per stream.
+ 
+## Static schemas
```
[the remark this answers is not in the document]
Comment (andnig) [reply airbyte-integrations/connectors/source-senseforce/source_senseforce/schemas/TODO.md]: Done

**Comment by andnig** [comment]
> Some comments, do you have test account to share to run tests?

@marcosmarxm Removed the files and answered your comments.
I sent you the credentials for a sandbox dataset in Slack on the 1st of November. My Slack Name is "Andreas Nigg". Are these credentials sufficient?

**Review by marcosmarxm — APPROVED** [review]
Thanks @andnig I'll publish your connector later!

**Pull request merged**